device/dhcp/servers/v6/dhcpv6server.yaml

```diff
@@ -0,0 +1,199 @@
+            Each IA holds one type of lease, like an identity association for temporary addresses (IA_TA) holds 
+            temporary addresses, and an identity association for prefix delegation (IA_PD).
+          type: string
+          default: iana
+          x-field-uid: 1
+          x-enum:
+            iana:
+              x-field-uid: 1
+            iata:
+              x-field-uid: 2
+            iapd:
+              x-field-uid: 3
+            ianapd:
+              x-field-uid: 4
+        iana:
+          $ref: '#/components/schemas/Dhcpv6Server.PoolInfo'
+          x-field-uid: 2
+        iata:
+          $ref: '#/components/schemas/Dhcpv6Server.PoolInfo'
+          x-field-uid: 3  
+        iapd:
+          $ref: '#/components/schemas/Dhcpv6Server.PrefixPoolInfo'
+          x-field-uid: 4
+        ianapd:
+          $ref: '#/components/schemas/Dhcpv6Server.PoolInfo'
+          x-field-uid: 5
+
+    Dhcpv6Server.PoolInfo:
+      description: >-
+        The container for pool configurations for IA types iana,iata and ianapd.
+      type: object
+      properties:
+        start_address:
+          description: >-
+            The IP address of the first lease pool.
+          type: string
+          format: ipv6
+          default: ::a0a:101
+          x-field-uid: 1
+        prefix:
+          description: >-
+            The IPv6 network prefix length to be applied to the address. 
+          type: integer
+          format: uint32
+          default: 64
+          maximum: 128
+          x-field-uid: 2
+        size:
+          description: >-
+            The total number of addresses in the pool.
+          type: integer
+          format: uint32
+          default: 10
+          minimum: 1
+          maximum: 1000000
+          x-field-uid: 3
+        step:
+          description: >-
+            The increment value for the lease address within the lease pool where multiple
+            addresses are present. The value is incremented according to the Prefix
+            Length and Step.
+          type: string
+          format: ipv6
+          default: 0::1
+          x-field-uid: 4
+
+    Dhcpv6Server.PrefixPoolInfo:
+      description: >-
+        The container for prefix pool configurations for IA type iapd.
+      type: object
+      properties:
+        start_prefix_address:
+          description: >-
+            The IP address of the first prefix pool.
+          type: string
+          format: ipv6
+          default: a1a1::0
+          x-field-uid: 1
+        prefix:
+          description: >-
+            The IPv6 network prefix length to be applied to the address. 
+          type: integer
+          format: uint32
+          default: 64
+          maximum: 128
+          x-field-uid: 2
+        size:
+          description: >-
+            The total number of addresses in the pool.
+          type: integer
+          format: uint32
+          default: 10
```

Comment:
Similar comments for default/max as previous PoolInfo structure .

Reply:
max can b added in v4 but step will be in v6 fomat as it is prefix

Reply:
updated step